### PR TITLE
chore: Add perf test

### PR DESCRIPTION
**Pull request by dvandra**
https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/138/files#diff-2d06c31adb3b8761408aedc8780c799d

### CODE REVIEW

**Comment by dvandra** [comment]
https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/138/files#diff-2d06c31adb3b8761408aedc8780c799d

File: src/perf.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+'use strict';
```
Comment (arajkumar) [inline src/perf.ts:1]: Please make the perf.ts as a standalone nodejs executable.
`node perf.ts`

**Review by arajkumar — CHANGES_REQUESTED** [review]
Remove all unrelated changes and make perf.ts as standlone node js executable.
`node perf.ts`

File: src/perf.ts (hunk 1 above)
Comment (dvandra) [reply src/perf.ts:1]: Ack.

File: src/performance.ts [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+async function runPerformance() {
+
+    let perf = new PerformanceComparision;
+
+    console.log("\n\n Comparision for 6 Dependencies: \n");
+    let payload6npm = [{"package":"lodash","version":"4.17.4"},{"package":"mongoose","version":"4.8.6"},{"package":"express-fileupload","version":"0.0.5"},{"package":"dustjs-linkedin","version":"2.5.0"},{"package":"adm-zip","version":"0.4.16"},{"package":"st","version":"2.0.0"}];
+    await perf.performanceTest("npm", payload6npm);
+    setTimeout(() => {}, 2000);
```
Comment (arajkumar) [inline src/performance.ts:84]: What is the use of the setTimeout?

File: src/performance.ts [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        }
+    }
+}
+
+async function runPerformance() {
+
+    let perf = new PerformanceComparision;
+
+    console.log("\n\n Comparision for 6 Dependencies: \n");
+    let payload6npm = [{"package":"lodash","version":"4.17.4"},{"package":"mongoose","version":"4.8.6"},{"package":"express-fileupload","version":"0.0.5"},{"package":"dustjs-linkedin","version":"2.5.0"},{"package":"adm-zip","version":"0.4.16"},{"package":"st","version":"2.0.0"}];
```
Comment (arajkumar) [inline src/performance.ts:82]: Can these payloads read from file rather than embedded inside source?

Comment (arajkumar) [reply src/performance.ts:82]: Having payload separate could also useful to run perf test using tools like ab(apache bench), wrk, jmeter

File: src/performance.ts [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+            console.log("\n Time taken for " + payload.length + " depensencies: [Normal call time = " + gTime + " ms]");
+        });
+    }
+
+    performanceTest = async (ecosystem, payload) => {
+        const iterationCount = 10;
+        for (let i = 0; i < iterationCount; i++) {
+            console.log("\n iteration count:" + (i+1));
+            await this.getTime(ecosystem, payload);
+            setTimeout(() => {}, 2000);
```
Comment (arajkumar) [inline src/performance.ts:70]: Ditto, what is the use of setTimeout?

**Review by arajkumar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**